components/collective-page/CollectiveNotificationBar.js

```diff
@@ -45,6 +50,12 @@ const messages = defineMessages({
 });
 
 const getNotification = (intl, status, collective, host) => {
+  if (status === 'collectiveCreated' && collective.isApproved) {
```

Comment:
Really small change, but I think this would be better to move that in the switch statement below.